clients/cobol-lsp-vscode-extension/src/services/snippetcompletion/cobolSnippets.json

```diff
@@ -96,55 +203,197 @@
     "prefix": "CALL",
     "body": [
       "CALL \"$1\" USING",
-      "    BY ${2|value,reference,content|} ${3:identifier}",
-      "    RETURNING ${4:return-code}",
+      "\tBY ${2|VALUE,REFERENCE,CONTENT|} ${3:identifier}",
+      "\tRETURNING ${4:return-code}",
       "END-CALL",
       "$0"
     ],
-    "description": "CALL literal"
+    "description": "The CALL statement transfers control from one compiled program to another. Note that the BY and RETURNING clauses are optional. For more on CALL: https://www.ibm.com/docs/en/cobol-zos/6.4?topic=statements-call-statement "
   },
   "CANCEL": {
     "prefix": "CANCEL",
-    "body": ["CANCEL ${1:id}", "$0"],
-    "description": "CANCEL literal"
+    "body": ["CANCEL ${1:id}.", "$0"],
+    "description": "The CANCEL statement terminates the subprogram. This ensures that data in the subprogram reverts to its initial state the next time the subprogram is called. Also when a CANCEL statement is executed all called programs contained within the program referenced in the CANCEL statement are canceled. https://www.ibm.com/docs/en/SS6SG3_6.4.0/pdf/pgmvs.pdf"
+  },
+  "CHARACTERS": {
+    "prefix": "CHARACTERS",
+    "body": [
+      "ENVIRONMENT DIVISION.",
+      "CONFIGURATION SECTION.",
+      "SYMBOLIC ${1|CHARACTER,CHARACTERS|} ${2:symbolic-character}"
+    ],
+    "description": "SYMBOLIC CHARACTER"
+  },
+  "CLASS DIGIT IS": {
+    "prefix": "CLASS",
+    "body": ["CLASS DIGIT IS \"0\" THROUGH \"9\""],
+    "description": ""
   },
   "CLOSE": {
     "prefix": "CLOSE",
     "body": ["CLOSE $(1:fileName}"],
     "description": "CLOSE filename"
   },
+  "CODE SET": {
+    "prefix": "CODE-SET",
+    "body": [
+      "DATA DIVISION.",
+      "FILE SECTION.",
+      "FD CODE-SET IS ${1:ALPHABET-VAR-NM}.",
+      "$0"
+    ],
+    "description": "CODE-SET is used in the COBOL DATA DIVISION, FILE SECTION, FD CLAUSE. https://www.ibm.com/docs/en/SS6SG3_6.4.0/pdf/pgmvs.pdf"
```

Comment:
Global: Can we be consistent about what we call the file FD / FD entry / FD clause? I would prefer "FD entry", "clause" is fine but the IBM documentation uses "entry". "File FD" is tautological.